up42/utils.py

```diff
@@ -270,22 +275,22 @@ def any_vector_to_fc(
         return fc
 
 
-def validate_fc_up42_requirements(fc: Union[dict, FeatureCollection]):
+def validate_fc_up42_requirements(fc: Union[dict, geojson.FeatureCollection]):
     """
     Validate the feature collection if it fits UP42 geometry requirements.
     """
     geometry_error = "UP42 only accepts single geometries, the provided geometry {}."
     if len(fc["features"]) != 1:
-        logger.info(geometry_error.format("contains multiple geometries"))
+        logger.info("%s contains multiple geometries", geometry_error)
         raise ValueError(geometry_error.format("contains multiple geometries"))
 
     fc_type = fc["features"][0]["geometry"]["type"]
     if fc_type != "Polygon":
-        logger.info(geometry_error.format(f"is a {fc_type}"))
+        logger.info("%s is a %s", geometry_error, fc_type)
```

Comment:
logger.error

Reply:
Why do we even log here and above if we raise an error with the same message?

Reply:
valid point.
I will remove the log.